db/import_mb_data.py

```diff
@@ -48,12 +66,12 @@ def load_artist_credit(connection, gids_in_AB, MB_release_data):
     return MB_artist_credit_data
 
 
-def load_artist_type(connection, gids_in_AB):
+def load_artist_type(connection, gids_in_AB, artist_credit_from_recording):
```

Comment:
Is the var `gids_in_AB` still needed here?

Reply:
No, not needed and I've removed it.